homeassistant/components/reisingerbridge/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "reisingerbridge",
+  "name": "Reisinger Bridge",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/reisingerbridge",
+  "requirements": ["reisinger-drive==0.2.0"],
```

Comment:
I cannot find the public repository providing the source code for this package. The PyPi listing also links to a 404. Could you point me to it?